R/fct_data_helpers.R

```diff
@@ -542,10 +542,10 @@ datatable_custom <- function(
     rename_vars = NULL, 
     title = NULL, 
     selection = "single",
-    extensions = "Scroller",
+    extensions = c("Scroller", "ColReorder"),
     plugins = "scrollResize",
     dom = "fti",
-    options = list(),
+    options = list(colReorder = TRUE),
     ...
```

Comment:
@aclark02-arcus I think it is probably better to add this to the list of modifyable defaults and add it to the function's description. @jthompson-arcus what do you think? In addition, can you also update the package documentation and include it in the PR?

Reply:
Yes, with how this function is currently setup, default options should be included in `default_opts`, not the options argument.